src/main/java/todolist/ToDoList.java

```diff
@@ -0,0 +1,239 @@
+     * @param task the Task to be added
+     */
+    public void addTask(Task task) {
+        this.toDoListArray.add(task);
+        System.out.println("added: " + task.getName());
+        System.out.println("Now you have " + this.toDoListArray.size() + " task(s) in the list");
+    }
+
+    public void deleteTask(int index) {
+        try {
+            String taskNameString = index + ". " + this.toDoListArray.get(index - 1).toString();
+            this.toDoListArray.remove(index - 1);
+            System.out.println("Noted. I've removed this task: ");
+            System.out.println(taskNameString);
+            System.out.println("Now you have " + this.toDoListArray.size() + " task(s) in the list");
+        } catch (IndexOutOfBoundsException e) {
+            ErrorMessages.taskDoesNotExistErrorMessage();
+        }
+    }
+
+    /**
+     * Get the string representation of the ToDoList
+     * @return the string representation of the ToDoList
+     */
+    @Override
+    public String toString() {
+        if (this.toDoListArray.isEmpty()) {
+            return "Empty ToDoList";
+        }
+        String output = "";
+        int count = 1;
+        for (Task item : this.toDoListArray) {
+            output += count + "." + item.toString() + "\n";
+            count++;
+        }
+        return output.substring(0, output.length()-1);
+    }
+
+    /**
+     * Print the feature intro message
+     */
+    public static void featureIntroMessage() {
+        System.out.println("[list] Show your Todolist tasks");
+        System.out.println("[mark X] Mark number X task as Done");
+        System.out.println("[unmark X] Mark number X task as UnDone");
+        System.out.println("[delete X] Delete number X task");
+        System.out.println("[todo XXX] Add a todo task");
+        System.out.println("[deadline XXX /by XXX] Add a deadline task");
+        System.out.println("[event XXX /from XXX /to XXX] Add a Event task");
+        System.out.println("[help] Show commands");
+        System.out.println("[quit] Return to main menu");
+        System.out.println("Where XXX Stands for any string (except space character)");
+        messageDivider();
+        System.out.println("Please enter a command:");
+    }
+
+    /**
+     * Print the message divider
+     */
+    public static void messageDivider() {
+        System.out.println("------------------------------------------");
+    }
+
+    private void writeToFile() throws IOException {
+        FileWriter fw = new FileWriter("ChatBBTData.txt");
+        fw.write(storeDataString());
+        fw.close();
+    }
+
+    private String storeDataString() {
+        StringBuilder dataString = new StringBuilder();
+        for (Task task : this.toDoListArray) {
+            dataString.append(task.storeDataString()).append("\n");
+        }
+        dataString.append("#end");
+        return dataString.toString();
+    }
+
+    public void loadData() {
+        String filePath = "ChatBBTData.txt";
```

Comment:
This seems to be used across files, maybe you could consider creating a class to contain constants that the program shares across the different source files?